templates/view_base.html

```diff
@@ -47,11 +47,7 @@
           <span class="icon-bar"></span>
           <span class="icon-bar"></span>
         </button>
-        {% if multice %}
-        <a href="/" class="navbar-brand">Grid Overview</a>
-        {% else %}
-        <a href="/" class="navbar-brand">HTCondor-CE Overview</a>
-        {% endif %}
+        <a href="/" class="navbar-brand">{{ self.title() }}</a>
```

Comment:
so much nicer...

Reply:
Yeah, it's crazy what nuggets I find when I actually read the documentation :D